source/content/drush-versions.md

```diff
@@ -54,21 +60,22 @@ Always use Drush 8 with Drupal 7 and Drupal 6 sites, as Drush 9 only works on Dr
 | Drush 7       | PHP 5.3.0+          |
 | Drush 8       | PHP 5.4.5+          |
 | Drush 9       | PHP 5.6.0+          |
+| Drush 10      | PHP 7.1.+           |
 
 See our guide on [Upgrading PHP Versions](/php-versions/).
 
 ## Troubleshooting
 
-Sometimes even after updating the drush version in `pantheon.yml`, the correct version of drush is not called. This is usually caused by an outdated configuration file, `policy.drush.inc`, in your local `~/.drush` directory, overriding `pantheon.yml`. Remove the file, or comment out its contents, to resolve.
+Sometimes even after updating the Drush version in `pantheon.yml`, the correct version of Drush is not called. This is usually caused by an outdated configuration file, `policy.drush.inc`, in your local `~/.drush` directory, overriding `pantheon.yml`. Remove the file, or comment out its contents to resolve.
```

Comment:
This can also be the case if a site-local version of Drush is present. We should note that site-local Drush is always used in preference to the Drush specified in pantheon.yml.